Jenkinsfile

```
@@ -80,12 +80,20 @@ node {
         try {
             sh "scripts/ci-setup --build"
             sh "scripts/ci-tests"
-        } finally {
-            sh "docker-compose down --volumes --remove-orphans"
+
             notify_slack([
+              stage: "Running test on tag ${tag}",
+              status: 'success'
+            ])
+
+        } catch (err) {
+          notify_slack([
```

Add 2 spaces to indentation.
```suggestion
            notify_slack([
```

---

Fixed in ac7272507f664aed0c3dcf0e5cc677dc41e8c609